packages/base/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@botpress/messaging-base",
+  "version": "0.0.1",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "source": "src/index.ts",
+  "license": "AGPL-3.0",
+  "scripts": {
+    "build": "yarn --silent && tsc --build",
+    "watch": "yarn --silent && tsc --build --watch"
+  },
+  "files": [
+    "dist"
+  ],
+  "devDependencies": {},
+  "dependencies": {}
```

Comment:
Shouldn't this package depend on typescript?

Reply:
Typescript is a dev dependency of the whole workspace so no